src/SmartComponents/SystemAdvisories/SystemAdvisories.test.js

```diff
@@ -40,153 +41,67 @@ const mockState = {
 };
 
 const initStore = (state) => {
-    const customMiddleWare = () => next => action => {
-        useSelector.mockImplementation(callback => {
-            return callback({  SystemAdvisoryListStore: state });
-        });
-        next(action);
-    };
-
-    const mockStore = configureStore([customMiddleWare]);
+    const mockStore = configureStore([]);
     return mockStore({  SystemAdvisoryListStore: state });
 };
 
-let wrapper;
 let store = initStore(mockState);
-
 beforeEach(() => {
-    store.clearActions();
-    useSelector.mockImplementation(callback => {
-        return callback({ SystemAdvisoryListStore: mockState });
-    });
-    wrapper = mount(<Provider store={store}>
-        <Router><SystemAdvisories inventoryId='test' /></Router>
-    </Provider>);
-});
-
-afterEach(() => {
-    useSelector.mockClear();
+    render(<ComponentWithContext renderOptions={{ store }}>
+        <SystemAdvisories inventoryId='test' />
+    </ComponentWithContext>);
 });
 
 describe('SystemAdvisories.js', () => {
-    it('Should dispatch CHANGE_ADVISORY_LIST_PARAMS only once on load', () => {
-        const dispatchedActions = store.getActions();
-        expect(dispatchedActions.filter(item => item.type === 'CHANGE_SYSTEM_SYSTEMS_LIST_PARAMS')).toHaveLength(1);
-    });
-
-    it('Should dispatch expandAdvisoryRow action onCollapse', () => {
-        wrapper.find('TableView').props().onCollapse(null, 0, 'testValue');
-
-        const dispatchedActions = store.getActions();
-        expect(dispatchedActions.filter(item => item.type === 'EXPAND_SYSTEM_ADVISORY_ROW')).toHaveLength(1);
-        expect(dispatchedActions[1].payload).toEqual({
-            rowId: 'RHSA-2020:2774',
-            value: 'testValue'
-        });
-    });
-
-    it('Should collapse a row ', () => {
-        wrapper.find('TableView').props().onCollapse(null, 0, 'testValue');
+    it('should handle remediation', async() => {
```

Comment:
nit-nit-nit-pick:
```suggestion
    it('should handle remediation', async () => {
```

![Alt Text](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMHM3YmUwZ2x6dWZpdjU2cTh6N3U3ZHc5ZGN6Y3NhazBobDJhM2R4OCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/BoGCXylFFL2OcTCzeG/giphy.gif)

Doesn't your linter/prettier automatically format the code for you? 🤔